configs/spotbugs-exclude.xml

```diff
@@ -7,6 +7,11 @@
 		<Bug
 			pattern="DM_DEFAULT_ENCODING,SQL_NONCONSTANT_STRING_PASSED_TO_EXECUTE" />
 	</Match>
+	<Match>
```

Comment:
I think it would have been cleaner to make the `generateOnlyKnown` field a member variable. This would also make the code a bit more future-proof (e.g., in case we create multiple providers with different options).

Reply:
It seems to be used quite a lot. I copied this from Postgres. If you have a clean fix for it there, I can do it the same way in Mz.